src/components/spreadsheet/custom-columns/use-node-aliases.ts

```diff
@@ -36,7 +38,7 @@ export const useNodeAliases = () => {
         } else {
             setNodeAliases(undefined);
         }
-    }, [snackError, studyUuid]);
+    }, [snackError, studyUuid, treeNodes]);
```

Comment:
watching treeNodes is a bit too much for our needs. 

ex: any node built => we call 2 times GET /node-aliases for nothing + some effects are triggered

maybe we should use a new notification ALIAS_UPDATE ?

Reply:
The existing notifications on node deletion or node renaming are now used to update the node aliases.
So, these aliases are now updated only when a node is removed or when a node is renamed ...